### DISCUSSION TITLE
“Best practices for Chrome extensions”?

### DISCUSSION THREAD

**Discussion by bsktcase** [coedit/gcal-rainbow-surf#13] [archive]
![image](https://github.com/user-attachments/assets/d4120d58-483f-4e0b-9dc4-e4e46e224c31)

Do we know what this means? Is it fixable? 😱

**Comment by mathiasboehmer** [comment]
Thank you for sharing! In German, it doesn't show the second half of the sentence ("because it doesn't follow best practices").

I hope so much, the extension will stay alive. I really really love it 💔

**Comment by ezdub** [comment]
@bsktcase @Towelie90 hey forgot to circle back here and let you know this is resolved now, no longer shows. thanks for reporting it!

**Reply by mathiasboehmer** [reply]
Thank's a lot. I really love the extension!
